build/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.6.0-dev
```

Comment:
That's unfortunate but understandable. I changed the version to `1.5.1`.

Reply:
One thing I wanted to confirm here - if people get this latest version of `build` but not the latest `build_runner_core` (which would now be possible) - they will still get an exception when trying to read files outside of lib right? It would just be an `AssetNotFoundException` instead of an `InvalidInputException`?

Reply:
Yes exactly. In the old `build_runner_core` version, the non-lib files aren't considered by the `AssetTracker` at all. So one would definitely get an `AssetNotFoundException`.